src/app/(auth)/login/loginModal.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+import {
+  AuthButton,
+  AuthModalContainer,
+  AuthModalContent,
+  HeadingTypography,
+  ParaTypography,
+  StyledBackdrop,
+  StyledLinkOne,
+  StyledLinkTwo,
+  StyledTextField,
+} from '../authModel.style';
+
+interface LoginModalProps {
+  open: boolean;
+  handleClose: () => void;
+}
+
+const LoginModal: React.FC<LoginModalProps> = ({ open, handleClose }) => {
+  return (
+    <AuthModalContainer
+      open={open}
+      onClose={handleClose}
+      BackdropComponent={StyledBackdrop}
+      disableScrollLock={false}
+    >
+      <AuthModalContent>
+        <HeadingTypography variant='h6'>Login</HeadingTypography>
+        <StyledTextField
```

Comment:
This form is missing error values, please include those aswell since it makes for better UX.